unchanged:
--- a/tools/hciattach.h	2018-06-27 09:35:07.010857759 +0530
+++ b/tools/hciattach.h	2018-06-27 09:43:38.617835299 +0530
@@ -45,6 +45,7 @@
 #define HCI_UART_AG6XX	9
 #define HCI_UART_NOKIA	10
 #define HCI_UART_MRVL	11
+#define HCI_UART_IBS	12
 
 #define HCI_UART_RAW_DEVICE	0
 #define HCI_UART_RESET_ON_INIT	1
unchanged:
--- a/tools/hciattach.c	2018-06-27 09:35:49.686772572 +0530
+++ b/tools/hciattach.c	2018-06-27 09:42:12.430007730 +0530
@@ -1099,7 +1099,7 @@
 			FLOW_CTL, DISABLE_PM, NULL, ath3k_ps, ath3k_pm  },
 
 	/* QCA ROME */
-	{ "qca",    0x0000, 0x0000, HCI_UART_H4, 115200, 115200,
+	{ "qca",    0x0000, 0x0000, HCI_UART_IBS, 115200, 115200,
 			FLOW_CTL, DISABLE_PM, NULL, qca, NULL },
 
 	/* QUALCOMM BTS */
only in patch2:
unchanged:
--- a/tools/hciattach_rome.h	2018-06-26 18:23:09.833006638 +0530
+++ b/tools/hciattach_rome.h	2018-06-27 09:38:29.006454382 +0530
@@ -269,8 +269,8 @@
 #else
 #define TF_RAMPATCH_TLV_1_0_0_PATH      "/lib/firmware/qca/rampatch_tlv_tf_1.0.tlv"
 #define TF_NVM_TLV_1_0_0_PATH           "/lib/firmware/qca/nvm_tlv_tf_1.0.bin"
-#define TF_RAMPATCH_TLV_1_0_1_PATH      "/lib/firmware/qca/tfbtfw11.tlv"
-#define TF_NVM_TLV_1_0_1_PATH           "/lib/firmware/qca/tfbtnv11.bin"
+#define TF_RAMPATCH_TLV_1_0_1_PATH      "/lib/firmware/qca/rampatch_tlv_tf_1.1.tlv"
+#define TF_NVM_TLV_1_0_1_PATH           "/lib/firmware/qca/nvm_tlv_tf_1.1.bin"
 #endif
 
 /* This header value in rampatch file decides event handling mechanism in the HOST */
